packages/wp-dashboard/src/components/editorSettings/shopping/shopify/index.js

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState, useCallback, useEffect } from '@googleforcreators/react';
+import PropTypes from 'prop-types';
+import { __, sprintf, TranslateWithMarkup } from '@googleforcreators/i18n';
+import { trackClick } from '@googleforcreators/tracking';
+import {
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  THEME_CONSTANTS,
+} from '@googleforcreators/design-system';
+
+/**
+ * Internal dependencies
+ */
+import {
+  InlineForm,
+  InlineLink,
+  SaveButton,
+  SettingsTextInput,
+  TextInputHelperText,
+  VisuallyHiddenLabel,
+} from '../../components';
+
+export const TEXT = {
+  HOST_CONTEXT: sprintf(
+    /* translators: 1: first example. 2: second example. */
+    __(
+      'Without <code>%1$s</code> or <code>%2$s</code>. Example: %3$s. <a>Find your URL</a>.',
```

Comment:
Noting the `Without` here is easy to miss when cutting and pasting a store name --- fine for this PR but should add validation around this.

Reply:
There is #11157 for this